x-pack/plugins/fleet/public/components/agent_enrollment_flyout/hooks.tsx

```diff
@@ -107,3 +135,13 @@ const getCloudSecurityIntegrationTypeFromPackagePolicy = (
   return packagePolicy?.inputs?.find((input) => input.enabled)
     ?.policy_template as CloudSecurityIntegrationType;
 };
+
+const getCloudSecurityIntegrationVersionFromAgentPolicy = (
+  agentPolicy?: AgentPolicy
+): string | undefined => {
+  const packagePolicy = agentPolicy?.package_policies?.find(
+    (input) => input.package?.name === FLEET_CLOUD_SECURITY_POSTURE_PACKAGE
+  );
+  if (!packagePolicy) return undefined;
+  return packagePolicy?.package?.version;
+};
```

Comment:
we can remove the early return since packagePolicy already uses optional chaining